modules/ROOT/pages/resource-usage.adoc

```diff
@@ -0,0 +1,28 @@
+= View Runtime Fabric Status and Resources
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can use Anypoint Runtime Manager to view the resource usage and health of your Anypoint Runtime Fabric clusters.
+
+== View Resource Usage 
```

Comment:
I suggest removing this flow completely given we don't report resource usage in the platform.

Reply:
I left the steps but removed the heading - do you think I should remove the first set of steps?